config/article_scrape_patterns.yml

```diff
@@ -27,9 +27,36 @@ presets:
       method: "xpath"
       pattern: "//meta[@property='article:section']/@content"
   time:
-    published: &published_time
+    article_date_original: &article_date_original
+      method: xpath
+      pattern: //meta[@namy='article_date_original']/@content
+    article_published_time: &article_published_time
       method: "xpath"
       pattern: "//meta[@property='article:published_time']/@content"
+    date: &date
+      method: xpath
+      pattern: //meta[@name='date']/@content
+    date_published: &date_published
+      method: xpath
+      pattern: //meta[@itemproy='datePublished']/@datetime
+    og_published_time: &og_published_time
+      method: xpath
+      pattern: //meta[@property='og:published_time']/@content
+    original_publication_date: &original_publication_date
+      method: xpath
+      pattern: //meta[@namy='OriginalPublicationDate']/@content
+    publication_date: &publication_date
+      method: xpath
+      pattern: //meta[@namy='publication_date']/@content
+    publish_date: &publish_date
+      method: xpath
+      pattern: //meta[@namy='PublishDate']/@content
+    rnews_datePublished: &rnews_datePublished
+      method: xpath
+      pattern: //meta[@property='rnews:datePublished']/@content
+    sailthru_date: &sailthru_date
+      method: xpath
+      pattern: //meta[@namy='sailthru.date']/@content
```

Comment:
Ive seen it on many sites, I'm going to research what it is, but it seems to be worth it